tests/spyre_util.py

```diff
@@ -197,14 +197,12 @@ def get_spyre_model_dir_path() -> Path:
     return Path(model_dir_path)
 
 
-# get model backends from env or default to all and add pytest markers
+# add pytest markers to supported different backends
 def get_spyre_backend_list():
-    user_backend_list = os.environ.get("VLLM_SPYRE_TEST_BACKEND_LIST",
-                                       "eager,inductor,sendnn")
+    backend_list = ["eager", "inductor", "sendnn"]
```

Comment:
I guess I could simplify this function more, but it does feel more readable this way 🤷